turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSwipeRefreshLayout.java

```diff
@@ -0,0 +1,28 @@
+package com.basecamp.turbolinks;
+
+import android.content.Context;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.util.AttributeSet;
+
+public class TurbolinksSwipeRefreshLayout extends SwipeRefreshLayout {
+    private CanScrollUpCallback callback;
+
+    public TurbolinksSwipeRefreshLayout(Context context) {
+        super(context);
+    }
+
+    public TurbolinksSwipeRefreshLayout(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    @Override
+    public boolean canChildScrollUp() {
+        if (callback != null) {
```

Comment:
I'd be OK making this `if` block a single-line expression. We've started to sway that direction a little more lately.